Infrastructure/gpg-ipdeny/deploy.sh

```diff
@@ -43,16 +43,16 @@ if [ -z "${PROTECTED_APP_SPACE_NAME+set}" ]; then
   exit 1
 fi
 
-if [ -z "${DENIED_IPS+set}" ]; then
-  echo "Must provide DENIED_IPS parameter -i"
+if [ -z "${DENIED_IPS_FILENAME+set}" ]; then
+  echo "Must provide DENIED_IPS_FILENAME parameter -f"
   exit 1
 fi
 
-IFS="," read -ra IPS <<< "$DENIED_IPS"
+readarray -t IPS < "$DENIED_IPS_FILENAME"
 
 NGINX_DENY_STATEMENTS=""
 for addr in "${IPS[@]}";
-  do NGINX_DENY_STATEMENTS="$NGINX_DENY_STATEMENTS deny $addr;"; true;
+  do NGINX_DENY_STATEMENTS="$NGINX_DENY_STATEMENTS deny ${addr//[$'\r']};"; true;
```

Comment:
Oh yeah I like this way more than commas

Reply:
Is it worth windows-proofing? will you get any line ending issues?

Reply:
I did have some issues with the line ending hence the ${addr//[$'\r']}, I think the readarray was just splitting on and stripping the \n of the \r\n in the text file. If we find we do need to strip other stuff it should just be a case of adding them here

Reply:
That works for me, as long as there isn't something easy we should be doing to make it a bit more resilient